akka-testkit-typed/src/main/scala/akka/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -116,6 +116,14 @@ trait ActorTestKit {
   final def spawn[T](behavior: Behavior[T], name: String): ActorRef[T] =
     spawn(behavior, name, Props.empty)
 
+  /**
+   * Stop an actor previously created with spawn.
+   */
+  final def stop[T](actorRef: ActorRef[T]): Unit = {
```

Comment:
do we have javadsl for this?

Reply:
Not yet, I was using it in a test then changed the test so i'll remove this from the API for now.